### ISSUE TITLE
[BUG] role not (yet) stable on 22.04

### ISSUE THREAD

**Issue by mircsicz** [tribe29/ansible-collection-tribe29.checkmk#98]
Hi all,

I'm trying to setup a cme server on a 22.04 host and sadly got this message:

```
TASK [tribe29.checkmk.server : Preflight - Fail if target host is not stable for this Role.] *****************************
fatal: [ans-tst]: FAILED! => {"changed": false, "msg": "This role is not stable for the target operating system, Ubuntu 22."}

```

Is there anything I could do to give it a try anyways?

And if it fails what could I do to support in getting it to work?

**Comment by robin-tribe29** [comment]
Hi @mircsicz, this is a feature, preventing users from running the role against an untested distribution.
You can override this security mechanism by adding `Ubuntu 22` to the variable `checkmk_server_server_stable_os` in `defaults/main.yml`.
If you can verify proper functionality - which is likely - we do appreciate your pull request. :)

**Comment by robin-tribe29** [comment]
Implemented support in `devel`.
This issue will close, once it is merged into `main`.
